scripts/launch_validator_node.sh

```diff
@@ -27,26 +27,30 @@ function print_ok {
 function define_config_file {
   if [[ $1 == "testnet" ]]; then
     export CONFIG_FILE=".env.testnet"
-  elif [[ $1 == "testnet-dev" ]]; then
-    export CONFIG_FILE=".env.testnet.dev"
+  elif [[ $1 == "devnet" ]]; then
```

Comment:
Let's change the order here as well. From dev to main

Reply:
updated